source/MaterialXGenMdl/mdl/materialx/pbrlib_1_9.mdl

```diff
@@ -0,0 +1,372 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Would it be acceptable to use the latest MaterialX project copyright for this new file?  Here's the text of that copyright notice for reference:

```
//
// Copyright Contributors to the MaterialX Project
// SPDX-License-Identifier: Apache-2.0
//
```

Reply:
I need to check. I do think so but I want to make sure.

Reply:
Changing the license text seems to be a bit more complicated and we need to involve legal. Since that will take a while for sure I would postpone that change for now.